### DISCUSSION TITLE
Applying Chakra styles to only a subset of a site?

### DISCUSSION THREAD

**Discussion by Teddarific** [chakra-ui/chakra-ui#7313] [Q&A]
A bit of an odd case — I'm working on a Chrome Extension that is injecting React into websites. I'd like to use `Chakra` to style this React, however, it seems like Chakra applies default global styles if there is a provider at any level in the DOM tree.

For example,

```
<div>
    Hello
</div>
<ChakraProvider>
   <div />
</ChakraProvider>
```

The `Hello` is still affected by Chakra.  Is there a way to prevent Chakra from applying styles globally and only within providers?

Similar things have been asked in the past, but are still unanswered:

- https://github.com/chakra-ui/chakra-ui/discussions/5950
- https://github.com/chakra-ui/chakra-ui/discussions/5147
- https://github.com/chakra-ui/chakra-ui/discussions/3960

**Comment by 9sami** [comment]
Hello @Teddarific, did you find any solution?
